package.json

```diff
@@ -19,16 +19,19 @@
     "format": "prettier --write \"**/*.{js,ts,css,scss,html,vue}\"",
     "format:check": "prettier --check \"**/*.{js,ts,css,scss,html,vue}\"",
     "format:check:full": "prettier --check \"**/*.{js,ts,vue,scss,css,html}\"",
-    "test": "jest"
+    "test": "jest",
```

Comment:
Is it necessary to keep this dependency here?

Reply:
Good catch! But yeah even though I'm not using Jest for this front-end testing, we still have jest tests for some of the requirements functionality that runs with `npm run test` so I think we do still need this.